apps/api/e2e/tests/voting.test.ts

```diff
@@ -542,6 +552,59 @@ describe("Voting API", () => {
         (stateResponse as VotingStateResponse).votingState.info.hasVoted,
       ).toBe(true);
     });
+
+    test("should maintain hasVoted=true after competition closes", async () => {
```

Comment:
This wasn't called explicitly in the ticket #646, but the value of `hasVoted` was always set to false when a competition status equaled "ended".  This seemed related so I'm fixing here.